spec/index.bs

```diff
@@ -361,34 +358,36 @@ interface GPUAdapter {
 
 To get a {{GPUAdapter}}, use {{GPU/requestAdapter()}}.
 
+<dfn dictionary>GPURequestAdapterOptions</dfn>
+provides hints to the user agent indicating what
```

Comment:
```suggestion
Provides hints to the user agent indicating what
```

Reply:
"GPURequestAdapterOptions" is the beginning of the sentence here.